web/cypress/integration/route/create-edit-delete-route.spec.js

```diff
@@ -18,23 +18,25 @@
 
 context('Create and Delete Route', () => {
   const name = `routeName${new Date().valueOf()}`;
+  const newName = `newName${new Date().valueOf()}`;
+  const sleepTime = 100;
 
   beforeEach(() => {
     // init login
     cy.login();
   });
 
-  it('create route', () => {
+  it('should create route', () => {
```

Comment:
Why do we need "should" here?

Reply:
`it should create a route`

Reply:
refer to [mocha](https://mochajs.org/), [should.js](https://github.com/tj/should.js/)

Reply:
got it, many thx